gensim/models/nmf.py

```diff
@@ -321,6 +321,10 @@ def get_term_topics(self, word_id, minimum_probability=None):
             if word_coef >= minimum_probability:
                 values.append((topic_id, word_coef))
 
+        if self.normalize:
+            factor_sum = sum(factor for topic_id, factor in values)
+            values = [(topic_id, factor / factor_sum) for topic_id, factor in values]
```

Comment:
Can the resulting vector be empty, or sum to 0.0 (division by zero)?

Reply:
It's possible when the number of topics is larger than `1/minimum_probability`, which is 1e8 by default. Seems like it's not a plausible situation

Reply:
So the only way to get zero `factor_sum` is if there are 1e8 topics? That's no concern then.

I was more worried about `values` being, or some such.